src/webview-contract/webviewDefinitions/draft/draftValidate.ts

```diff
@@ -0,0 +1,19 @@
+import { WebviewDefinition } from "../../webviewTypes";
+
+export interface InitialState {
+    validationResults: string;
+}
+
+export type ExistingFiles = string[];
+
+export type ToVsCodeMsgDef = {
+    createDraftValidateRequest: string;
+};
+
+export type ToWebViewMsgDef = {
+    validationResult: { result: string };
+};
+
+export type DraftValidateDefinition = WebviewDefinition<InitialState, ToVsCodeMsgDef, ToWebViewMsgDef>;
+
+//TODO FINISH REPLACING UNUSED CODE
```

Comment:
💡 Please Note: ⛩️  Might be this is mistake but if there is something unused lets remove it or might be jsut mention why is is there so anyone notices can remove post this, if its not possible right now.

Reply:
Sorry, that was left over comment